<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="7" failed="1" total="9" passed="1">
  <reporter-output>
    <line>
      <![CDATA[[INFO]: The test Browser is FIREFOX !!!]]>
    </line>
    <line>
      <![CDATA[[INFO]:  The application url is :- http://qad.ng.cengage.com/static/nb/login.html]]>
    </line>
    <line>
      <![CDATA[[INFO]: log into the application using credentials 'automationadmin@qait.com / A111111']]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: PageTitle for NG-30181-AdminPage contains: 'MindTap Admin - Manage Masters'.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: Verified User is on Admin Page]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element orgaizationTab is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element mindtap_toolbar is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element admin_dashboard with text Admin Dashboard is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element inputOrganizationSearch is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element courseOrganization is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element courseProvisioned is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element courseCollege with text Stewart_Precalculus: Mathematics for Calculus_7e_MTQ is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element provisionAppsIcon is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element appRegistryHeading is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element appRegistryHeading is displayed.]]>
    </line>
    <line>
      <![CDATA[[INFO]: The Test: TC01_INSTRUCTOR_ADDS_OFFLINE_ACTIVITY COMPLETED!
]]>
    </line>
  </reporter-output>
  <suite name="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity" duration-ms="94116" started-at="2015-10-14T12:25:03Z" finished-at="2015-10-14T12:26:37Z">
    <groups>
    </groups>
    <test name="Command line test 33251e9b-aa27-4e67-9eb0-372293da7a1c" duration-ms="94116" started-at="2015-10-14T12:25:03Z" finished-at="2015-10-14T12:26:37Z">
      <class name="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity">
        <test-method status="PASS" signature="start_test_session(java.lang.String)[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="start_test_session" is-config="true" duration-ms="8139" started-at="2015-10-14T17:55:03Z" finished-at="2015-10-14T17:55:11Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[[INFO]: The test Browser is FIREFOX !!!]]>
            </line>
            <line>
              <![CDATA[[INFO]:  The application url is :- http://qad.ng.cengage.com/static/nb/login.html]]>
            </line>
          </reporter-output>
        </test-method> <!-- start_test_session -->
        <test-method status="PASS" signature="Step_05_Admin_Logs_in_to_the_Application()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="Step_05_Admin_Logs_in_to_the_Application" duration-ms="9754" started-at="2015-10-14T17:55:11Z" finished-at="2015-10-14T17:55:21Z">
          <reporter-output>
            <line>
              <![CDATA[[INFO]: log into the application using credentials 'automationadmin@qait.com / A111111']]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: PageTitle for NG-30181-AdminPage contains: 'MindTap Admin - Manage Masters'.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: Verified User is on Admin Page]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element orgaizationTab is displayed.]]>
            </line>
          </reporter-output>
        </test-method> <!-- Step_05_Admin_Logs_in_to_the_Application -->
        <test-method status="PASS" signature="takeScreenshotonFailure(org.testng.ITestResult)[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="takeScreenshotonFailure" is-config="true" duration-ms="1" started-at="2015-10-14T17:55:21Z" finished-at="2015-10-14T17:55:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Step_05_Admin_Logs_in_to_the_Application status=SUCCESS method=TC01_Instructor_Adds_Offline_Activity.Step_05_Admin_Logs_in_to_the_Application()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] output=[INFO]: log into the application using credentials 'automationadmin@qait.com / A111111']]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotonFailure -->
        <test-method status="FAIL" signature="Step_06_Admin_Search_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="Step_06_Admin_Search_Course" duration-ms="76071" started-at="2015-10-14T17:55:21Z" depends-on-methods="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.Step_05_Admin_Logs_in_to_the_Application" finished-at="2015-10-14T17:56:37Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 30 seconds waiting for visibility of [[FirefoxDriver: firefox on WINDOWS (80298a2a-cb1b-4bb7-9dcc-e79730e99649)] -> xpath: //div[contains(text(),'Cengage.non-mt-activity')]]
Build info: version: '2.46.0', revision: '61506a4624b13675f24581e453592342b7485d71', time: '2015-06-04 10:22:50'
System info: host: 'LNDA0021', ip: '10.0.9.31', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_51'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=41.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 80298a2a-cb1b-4bb7-9dcc-e79730e99649]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of [[FirefoxDriver: firefox on WINDOWS (80298a2a-cb1b-4bb7-9dcc-e79730e99649)] -> xpath: //div[contains(text(),'Cengage.non-mt-activity')]]
Build info: version: '2.46.0', revision: '61506a4624b13675f24581e453592342b7485d71', time: '2015-06-04 10:22:50'
System info: host: 'LNDA0021', ip: '10.0.9.31', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_51'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=41.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 80298a2a-cb1b-4bb7-9dcc-e79730e99649
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:229)
	at com.qait.mindtap.automation.utils.SeleniumWait.waitForElementToBeVisible(SeleniumWait.java:53)
	at com.qait.mindtap.automation.getpageobjects.GetPage.element(GetPage.java:136)
	at com.qait.mindtap.automation.getpageobjects.GetPage.element(GetPage.java:130)
	at com.qait.mindtap.automation.getpageobjects.GetPage.isElementDisplayed(GetPage.java:274)
	at com.qait.mindtap.keywords.NG_30181_AdminPageActions.selectCengageNoMT_Activity(NG_30181_AdminPageActions.java:79)
	at com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.Step_06_Admin_Search_Course(TC01_Instructor_Adds_Offline_Activity.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
            <line>
              <![CDATA[[ASSERT PASS]: element mindtap_toolbar is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element admin_dashboard with text Admin Dashboard is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element inputOrganizationSearch is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element courseOrganization is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element courseProvisioned is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element courseCollege with text Stewart_Precalculus: Mathematics for Calculus_7e_MTQ is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element provisionAppsIcon is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element appRegistryHeading is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element appRegistryHeading is displayed.]]>
            </line>
          </reporter-output>
        </test-method> <!-- Step_06_Admin_Search_Course -->
        <test-method status="PASS" signature="takeScreenshotonFailure(org.testng.ITestResult)[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="takeScreenshotonFailure" is-config="true" duration-ms="3" started-at="2015-10-14T17:56:37Z" finished-at="2015-10-14T17:56:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Step_06_Admin_Search_Course status=FAILURE method=TC01_Instructor_Adds_Offline_Activity.Step_06_Admin_Search_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] output=[ASSERT PASS]: element mindtap_toolbar is displayed.]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotonFailure -->
        <test-method status="SKIP" signature="step_07_Instructor_Logs_Again(java.lang.String)[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="step_07_Instructor_Logs_Again" duration-ms="0" started-at="2015-10-14T17:56:37Z" depends-on-methods="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.Step_06_Admin_Search_Course" finished-at="2015-10-14T17:56:37Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC01_Instructor_Adds_Offline_Activity.step_07_Instructor_Logs_Again(java.lang.String)[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC01_Instructor_Adds_Offline_Activity.step_07_Instructor_Logs_Again(java.lang.String)[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- step_07_Instructor_Logs_Again -->
        <test-method status="SKIP" signature="Step_08_Instructor_Logs_in_to_the_Application()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="Step_08_Instructor_Logs_in_to_the_Application" duration-ms="0" started-at="2015-10-14T17:56:37Z" depends-on-methods="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.step_07_Instructor_Logs_Again" finished-at="2015-10-14T17:56:37Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC01_Instructor_Adds_Offline_Activity.Step_08_Instructor_Logs_in_to_the_Application()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC01_Instructor_Adds_Offline_Activity.Step_08_Instructor_Logs_in_to_the_Application()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_08_Instructor_Logs_in_to_the_Application -->
        <test-method status="SKIP" signature="Step_09_Instructor_Manages_Course_From_SSO()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="Step_09_Instructor_Manages_Course_From_SSO" duration-ms="0" started-at="2015-10-14T17:56:37Z" depends-on-methods="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.Step_08_Instructor_Logs_in_to_the_Application" finished-at="2015-10-14T17:56:37Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC01_Instructor_Adds_Offline_Activity.Step_09_Instructor_Manages_Course_From_SSO()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC01_Instructor_Adds_Offline_Activity.Step_09_Instructor_Manages_Course_From_SSO()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_09_Instructor_Manages_Course_From_SSO -->
        <test-method status="SKIP" signature="Step_11_Instructor_Enters_Detail_In_OfflineActivity()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="Step_11_Instructor_Enters_Detail_In_OfflineActivity" duration-ms="0" started-at="2015-10-14T17:56:37Z" depends-on-methods="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.Step_10_Verify_Clicking_Add_Takes_To_RollingWeekUI" finished-at="2015-10-14T17:56:37Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC01_Instructor_Adds_Offline_Activity.Step_11_Instructor_Enters_Detail_In_OfflineActivity()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC01_Instructor_Adds_Offline_Activity.Step_11_Instructor_Enters_Detail_In_OfflineActivity()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_11_Instructor_Enters_Detail_In_OfflineActivity -->
        <test-method status="SKIP" signature="Step_13_Verify_Activity_Gets_Added()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="Step_13_Verify_Activity_Gets_Added" duration-ms="0" started-at="2015-10-14T17:56:37Z" depends-on-methods="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.Step_12_Verify_Rolling_Week_UI" finished-at="2015-10-14T17:56:37Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC01_Instructor_Adds_Offline_Activity.Step_13_Verify_Activity_Gets_Added()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC01_Instructor_Adds_Offline_Activity.Step_13_Verify_Activity_Gets_Added()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_13_Verify_Activity_Gets_Added -->
        <test-method status="SKIP" signature="Step_12_Verify_Rolling_Week_UI()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="Step_12_Verify_Rolling_Week_UI" duration-ms="0" started-at="2015-10-14T17:56:37Z" depends-on-methods="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.Step_11_Instructor_Enters_Detail_In_OfflineActivity" finished-at="2015-10-14T17:56:37Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC01_Instructor_Adds_Offline_Activity.Step_12_Verify_Rolling_Week_UI()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC01_Instructor_Adds_Offline_Activity.Step_12_Verify_Rolling_Week_UI()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_12_Verify_Rolling_Week_UI -->
        <test-method status="SKIP" signature="Step_10_Verify_Clicking_Add_Takes_To_RollingWeekUI()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="Step_10_Verify_Clicking_Add_Takes_To_RollingWeekUI" duration-ms="0" started-at="2015-10-14T17:56:37Z" depends-on-methods="com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity.Step_09_Instructor_Manages_Course_From_SSO" finished-at="2015-10-14T17:56:37Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC01_Instructor_Adds_Offline_Activity.Step_10_Verify_Clicking_Add_Takes_To_RollingWeekUI()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC01_Instructor_Adds_Offline_Activity.Step_10_Verify_Clicking_Add_Takes_To_RollingWeekUI()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_10_Verify_Clicking_Add_Takes_To_RollingWeekUI -->
        <test-method status="PASS" signature="stop_test_session()[pri:0, instance:com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity@17f6480]" name="stop_test_session" is-config="true" duration-ms="106" started-at="2015-10-14T17:56:37Z" finished-at="2015-10-14T17:56:37Z">
          <reporter-output>
            <line>
              <![CDATA[[INFO]: The Test: TC01_INSTRUCTOR_ADDS_OFFLINE_ACTIVITY COMPLETED!
]]>
            </line>
          </reporter-output>
        </test-method> <!-- stop_test_session -->
      </class> <!-- com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity -->
    </test> <!-- Command line test 33251e9b-aa27-4e67-9eb0-372293da7a1c -->
  </suite> <!-- com.qait.mindtap.tests.NG_30370.TC01_Instructor_Adds_Offline_Activity -->
</testng-results>
